academy/3-my-own-chain/create-message.md

```diff
@@ -323,11 +323,11 @@ Usage:
 Flags:
    -a, --account-number uint      The account number of the signing account (offline mode only)
    -b, --broadcast-mode string    Transaction broadcasting mode (sync|async|block) (default "sync")
-       --dry-run                  ignore the --gas flag and perform a simulation of a transaction, but don't broadcast it
+       --dry-run                  Ignore the --gas flag and perform a simulation of a transaction, but don't broadcast it
```

Comment:
Actually, this is copied from a command output. So although it looks better capitalised, this is not what a student would see ;)
No need to change back.